docs/current/api/975146-concepts.md

```diff
@@ -0,0 +1,126 @@
+---
+slug: /api/concepts
+---
+
+# Key Concepts
+
+## Introduction
+
+One of Dagger's most powerful features is cross-language instrumentation: teams can use each other's tools without learning each other's language. Dagger achieves this by exposing its functionality through GraphQL, which serves as a low-level language-agnostic API framework for all Dagger operations.
+
+This approach has two main benefits:
+
+- It enables CI/CD pipeline authors to use whichever language feels natural and intuitive to them, without having to think about interoperability.
+- Rather than reimplementing the same automation in every possible programming language, it only needs to be implemented once and is then reusable by all. As a result, two cooperating teams no longer necessarily have to come to consensus on a common language.
+
+This tutorial explains the key concepts you need to use the Dagger GraphQL API.
+
+:::note
+This tutorial explains some basic GraphQL concepts in the context of the Dagger GraphQL API, but it is not intended as a full-featured GraphQL tutorial. To learn more about GraphQL and how it works, read the [GraphQL tutorial](https://graphql.org/learn/).
+:::
+
+## Objects, Fields and Build Operations
+
+Let's start with a basic query:
+
+```graphql
+query {
+  container {
+    from(address: "alpine:latest") {
+      exec(args: ["apk", "info"]) {
+        stdout {
+          contents
+        }
```

Comment:
```suggestion
        stdout
```

As of today's release the API for `stdout` has changed.

Reply:
Fixed